internal/pkg/describe/env.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"encoding/json"
+	//"encoding/json"
+	"fmt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+
+	//"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+)
+
+type Environment struct {
+	Name       string `json:"name"`
+	Production bool   `json:"production"` // add to instances used elsewhere (ie project_show)?
+	Region     string `json:"region"`
+	AccountID  string `json:"accountID"`
+}
+
+type EnvApp struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+type EnvTag struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type EnvSummary struct {
+	Environment  []*Environment `json:"environment"`
+	Applications []*EnvApp      `json:"applications"`
+	Tags         []*EnvTag      `json:"tags"`
```

Comment:
You may be able to just have a `map[string]string` here similar to how it's specified [here](https://github.com/aws/amazon-ecs-cli-v2/blob/master/internal/pkg/archer/project.go#L12) rather than have the `EnvTag` custom type.